Tests/CarthageKitTests/Resources/Resolver/ConflictingDependencies/Repository/dependencies/BMBiometrics/0.1.0.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "versionSpecifier" : "~> 1.0.0",
+    "dependency" : {
+      "git" : {
+        "urlString" : "https:\/\/git.behindmedia.com\/Common\/SwiftLint.git"
+      }
+    }
+  },
+  {
+    "versionSpecifier" : "~> 0.13.0",
+    "dependency" : {
+      "git" : {
+        "urlString" : "https:\/\/git.behindmedia.com\/Common\/BMFoundation.git"
+      }
+    }
+  }
```

Comment:
Where are all these fixtures used? Do we really need _all_ of them?

Reply:
All these fixtures are used in the pending test cases in ResolverSpec. They are direct results of using the diagnose command for concrete problematic dependency trees.

Reply:
Can we please leave any pending test cases and related fixtures for a separate PR?